interfaces/Portalicious/src/locale/messages.xlf

```diff
@@ -1586,7 +1586,10 @@
         <source>Add a reason</source>
       </trans-unit>
       <trans-unit id="3720159686083335313" datatype="html">
-        <source>You&apos;re about to mark <x equiv-text="{{ duplicatesRegistrationIds().length === 2 ? &apos;both&apos; : duplicatesRegistrationIds().length }}" id="INTERPOLATION"/> registrations as unique. This means that they will no longer appear as duplicates of each other.<x ctype="lb" equiv-text="&lt;br /&gt;" id="LINE_BREAK"/><x ctype="lb" equiv-text="&lt;br /&gt;" id="LINE_BREAK"/>However, if either is found to be a duplicate of other registrations, their duplication status will be &apos;Duplicate&apos;.</source>
+        <source>You&apos;re about to mark <x equiv-text="{{ duplicatesRegistrationIds().length | i18nPlural : { &apos;=2&apos;: &apos;both&apos;, other: duplicatesRegistrationIds().length.toString(), } }}" id="INTERPOLATION"/> registrations as unique. This means that they will no longer appear as duplicates of each other.<x ctype="lb" equiv-text="&lt;br /&gt;" id="LINE_BREAK"/><x ctype="lb" equiv-text="&lt;br /&gt;" id="LINE_BREAK"/>However, if either is found to be a duplicate of other registrations, their duplication status will be &apos;Duplicate&apos;.</source>
```

Comment:
Hmm.. I was hoping the Angular i18n-pipe would help with this output...

I don't think translators can us this easily... :/